packages/vue/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { createVueEventProcessor } from './eventprocessor';
```

Comment:
I really don't like that we have 2 separate indexes instead of just one with conditionals, as it's error prone and we have to keep 2 files in synce, but we already do this in other places :(